spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+include SessionsHelper
+
+RSpec.describe PostsController, type: :controller do
+  before do
+    Rails.application.load_seed
+  end
+
+  let(:user) { User.find_by(username: 'user1') }
+
+  context 'GET #new' do
```

Comment:
🤔  actually it would be more clear to structure it like this 

```
context "as logged in user"
  describe "GET #new"
  describe  "Post #create"
context "as guest"
  describe "GET #new"
  describe  "Post #create"
```